### PR TITLE
Documentation library

### PR DESCRIPTION
**Pull request by mtronci**
This pull requests contains a small change, aiming to bring the `TUTORIAL.MD` file in line with the Regulatory Reporting page on the [Google Cloud Architecture Centre](https://cloud.google.com/architecture/set-up-regulatory-reporting-architecture-bigquery#upload_the_sample_data).

### CODE REVIEW

File: use_cases/examples/bird/logs/dbt.log [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (t8dizzog) [inline use_cases/examples/bird/logs/dbt.log:1]: Do we want to check in this file?

**Comment by mescanne** [comment]
There's also the target/ sub-directory of dbt in there -- I'm not sure we want that as well? And the bird example?

File: use_cases/examples/eba_dpm/dbt/logs/dbt.log.legacy [hunk 2]
```diff
@@ -0,0 +1,31 @@
+2022-08-26 15:06:43.837253 (MainThread): 'soft_unicode' has been renamed to 'soft_str'. The old name will be removed in MarkupSafe 2.1.
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/logs/dbt.log.legacy:1]: Do we want to check in this file @mescanne @mtronci ?

File: use_cases/examples/eba_dpm/dbt/target/compiled/eba_dpm/models/example/my_second_dbt_model.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+-- Use the `ref` function to select from other models
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/target/compiled/eba_dpm/models/example/my_second_dbt_model.sql:1]: Do we want to check in this file @mescanne @mtronci ?

File: use_cases/examples/eba_dpm/dbt/target/compiled/eba_dpm/models/src/DataType.sql [hunk 4]
```diff
@@ -0,0 +1 @@
+SELECT * FROM `scannell-fsi-dev`.`dpm_model`.`v3_2_DataType`
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/target/compiled/eba_dpm/models/src/DataType.sql:1]: Do we want to check in this file @mescanne @mtronci ?

File: use_cases/examples/eba_dpm/dbt/target/compiled/eba_dpm/models/table_layout.sql [hunk 5]
```diff
@@ -0,0 +1 @@
+SELECT * FROM `scannell-fsi-dev`.`dpm_model`.`v3_2_DataType`
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/target/compiled/eba_dpm/models/table_layout.sql:1]: Do we want to check in this file @mescanne @mtronci ?

File: use_cases/examples/eba_dpm/dbt/target/run/eba_dpm/models/example/my_first_dbt_model.sql [hunk 6]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/target/run/eba_dpm/models/example/my_first_dbt_model.sql:1]: Do we want to check in this file @mescanne @mtronci ?

File: use_cases/examples/eba_dpm/dbt/target/run/eba_dpm/models/example/my_second_dbt_model.sql [hunk 7]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/target/run/eba_dpm/models/example/my_second_dbt_model.sql:1]: Do we want to check in this file @mescanne @mtronci ?

File: use_cases/examples/eba_dpm/dbt/target/run/eba_dpm/models/src/DataType.sql [hunk 8]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/target/run/eba_dpm/models/src/DataType.sql:1]: Do we want to check in this file @mescanne @mtronci ?

File: use_cases/examples/eba_dpm/dbt/target/run/eba_dpm/models/table_layout.sql [hunk 9]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/target/run/eba_dpm/models/table_layout.sql:1]: Do we want to check in this file @mescanne @mtronci ?

File: use_cases/examples/eba_dpm/dbt/target/run_results.json [hunk 10]
```diff
@@ -0,0 +1 @@
...t_version": "1.2.0", "generated_at": "2022-08-26T15:45:43.820845Z", "invocation_id": "7cba58ad-1bbd-4e3e-9c68-bbe0cb1be385", "env": {}}, "results": [{"status": "success", "timing": [{"name": "compile", "started_at": "2022-08-26T15:45:39.900328Z", "completed_at": "2022-08-26T15:45:39.910688Z"}, {"name": "execute", "started_at": "2022-08-26T15:45:39.911278Z", "completed_at": "2022-08-26T15:45:43.815586Z"}], "thread_id": "Thread-1 (worker)", "execution_time": 3.917415142059326, "adapter_response": {"_message": "CREATE TABLE (9.0 rows, 152.0 Bytes processed)", "code": "CREATE TABLE", "rows_affected": 9, "bytes_processed": 152}, "message": "CREATE TABLE (9.0 rows, 152.0 Bytes processed)", "failures": null, "unique_id": "model.eba_dpm.DataType"}], "elapsed_time": 6.961270809173584, "args": {"write_json": true, "use_colors": true, "printer_width": 100, "version_check": true, "partial_parse": false, "static_parser": true, "profiles_dir": "/Users/mtronci/Code/reg-reporting-blueprint/use_cases/examples/eba_dpm/dbt/profiles", "send_anonymous_usage_stats": false, "event_buffer_size": 100000, "quiet": false, "no_print": false, "which": "run", "rpc_method": "run", "indirect_selection": "eager"}}
```
Comment (t8dizzog) [inline use_cases/examples/eba_dpm/dbt/target/run_results.json:1]: Do we want to check in this file @mescanne @mtronci ?

**Pull request closed without merging**